client/src/featureform/enums.py

```diff
@@ -19,6 +34,21 @@ class ScalarType(Enum):
 
 
 class ResourceStatus(Enum):
+    """
```

Comment:
I am curious if we need this since users would not use this at all except for the status on the dashboard or status?

Reply:
Technically, no, but I figured it might be good if folks encounter the enums in code that is user-facing/public.